packages/block-editor/src/components/block-list/content.scss

```diff
@@ -416,6 +416,7 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	justify-content: center;
 	font-size: $default-font-size;
 	font-family: $default-font;
+	color: $black;
```

Comment:
This needs to be `$gray-900`.

https://github.com/WordPress/gutenberg/blob/93323fd1a3cecd0771ead22b16a9661c53be6212/packages/base-styles/_colors.scss#L8

Reply:
Or maybe you used `$black` because it _needed_ that level of contrast?

If so let's add a comment to explain why we're using it.

Reply:
It feels on that ray we need black.

Reply:
Yeh the contrast would be insufficient otherwise. Thanks for clarifying 👍